client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -652,6 +669,29 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
         return new ViewMotionChangeRecommendation(changeReco);
     }
 
+    /**
+     * Creates a {@link ViewMotionChangeRecommendation} object to change the title, based on the motion ID.
+     * This object is not saved yet and does not yet have any changed title. It's meant to populate the UI form.
+     *
+     * @param {number} motionId
+     * @param {number} lineLength
+     */
+    public createTitleChangeRecommendationTemplate(
```

Comment:
please use the `ChangeRecommendationRepositoryService` class or create a new service